independent-projects/tools/pom.xml

```diff
@@ -54,6 +54,7 @@
     <modules>
         <module>platform-descriptor-api</module>
         <module>common</module>
+        <module>qoder</module>
```

Comment:
let's probably come up with something less obscure

Reply:
changed to `quarkus-devtools-codegen`